week03_model_free/display_server_preparation.sh

```diff
@@ -0,0 +1,47 @@
+echo "apt-get updating..."
+apt-get -qq update > /dev/null
+echo "x11vnc installing if it is not exists..."
+echo ""
+apt-get -q install -y x11vnc
+echo ""
+echo "x11vnc is installed"
+checkXvfbProcess=1 # 0 == True, 1 == False
+if [ "$checkXvfbProcess" -eq 0 ];
+then
+    if [ -e ./xvfb.pid ];
+    then
+        xpid="$(cat ./xvfb.pid)"
+        com="$(ps -p ${xpid} -o comm=)"
+        idealcom="Xvfb"
+        if [ "$com" != "$idealcom" ];
```

Comment:
Can be simplified to `"$com" != 'Xvfb'`.

Reply:
Let it be some small self-documented piece of code.